engines/dradis-api/app/controllers/dradis/ce/api/v1/attachments_controller.rb

```diff
@@ -0,0 +1,101 @@
+module Dradis::CE::API
+  module V1
+    class AttachmentsController < Dradis::CE::API::V1::ProjectScopedController
+      before_action :set_node
+
+      skip_before_action :json_required, :only => [:create]
+
+      def index
+        @attachments = @node.attachments.each(&:close)
+      end
+
+      def show
+        filename = params[:id]
```

Comment:
You can replace the `:id` param name with `:filename` instead. `resources :attachments, param: :filename`